client/components/bancontactPayments/html/README.md

```diff
@@ -0,0 +1,170 @@
+   ```
+
+2. **Install dependencies:**
+
+   ```bash
+   npm install
+   ```
+
+3. **Start the development server:**
+   ```bash
+   npm start
+   ```
+   The demo will be available at `http://localhost:3000`
+
+## How It Works
+
+### Client-Side Flow
+
+1. **SDK Initialization**: Loads PayPal Web SDK with Bancontact components using a client token fetched from the server's `/paypal-api/auth/browser-safe-client-token` endpoint
+2. **Eligibility Check**: Verifies Bancontact is eligible for the merchant
+3. **Session Creation**: Creates Bancontact payment session with event callbacks for handling payment lifecycle events
+4. **Field Setup**: Mounts the required full name field
+5. **Validation**: Validates fields before initiating the payment flow
+6. **Payment Flow**: Opens a popup window where customers select their bank and authorize the payment. The order is created server-side via `/paypal-api/checkout/orders/create` before displaying the popup
+7. **Completion**: Processes the payment result by capturing the approved order via `/paypal-api/checkout/orders/:orderId/capture`, or handles cancellation and error scenarios appropriately
+
+### Server-Side Requirements
+
+The integration requires these endpoints (provided by the API server):
+
+- `GET /paypal-api/auth/browser-safe-client-token` - Generate client token
+- `POST /paypal-api/checkout/orders/create` - Create order
+- `POST /paypal-api/checkout/orders/:orderId/capture` - Capture order
+
+## Available Scripts
+
+- `npm start` - Start Vite development server (port 3000)
+- `npm run build` - Build for production
+- `npm run preview` - Preview production build
+- `npm run format` - Format code with Prettier
+- `npm run format:check` - Check code formatting
+
+## Troubleshooting
+
+### Bancontact not eligible
+
+- Verify `testBuyerCountry` is set to "BE"
+- Check that Bancontact is enabled for the merchant in PayPal account settings
+
+### Validation fails
+
+- Ensure full name field is properly mounted
+- Check that field has valid input
+- Verify field is visible in the DOM
+
+### Payment popup doesn't open
+
+- Check for popup blockers
+- Verify order creation returns valid orderId
+- Ensure proper event handler setup
+- Check browser console for errors
+
+### Order creation fails
+
+- Verify API server is running on port 8080
+- Check server logs for errors
+- Validate order payload format
+
+## Key Differences from Other Payment Methods
+
+- **Currency**: Bancontact requires EUR (Euro) as the currency
+- **Country**: Primarily used in Belgium (set `testBuyerCountry: "BE"`)
+- **Required Field**: Full name field is mandatory for Bancontact payments
+- **Presentation Mode**: Supports popup mode for the payment flow
+
+## Documentation
+
+- [PayPal Developer Documentation](https://developer.paypal.com/docs/)
+- [PayPal Developer Community](https://developer.paypal.com/community/)
+- [Bancontact Integration Guide](../../../BANCONTACT_INTEGRATION_GUIDE.md)
```

Comment:
I don't see this file in the repo. Is this something that will be added in another PR? What will this file contain outside of what's in this README already?